compositor_web/src/wasm.rs

```diff
@@ -117,4 +120,16 @@ impl LiveCompositorRenderer {
             .register_renderer(RendererId(renderer_id.into()), renderer_spec)
             .map_err(to_js_error)
     }
+
+    pub async fn register_font(&mut self, font_url: String) -> Result<(), JsValue> {
+        let text_renderer_ctx = self.renderer.text_renderer_ctx();
+        let bytes = download(&font_url).await?;
+        text_renderer_ctx.add_font(Source::Binary(Arc::new(bytes.to_vec())));
+        Ok(())
+    }
+}
+
+async fn download(url: &str) -> Result<Bytes, JsValue> {
+    let resp = reqwest::get(url).await.map_err(to_js_error)?;
```

Comment:
what about error handling, if you get 404 you will try to parse some error message as font